features/step_definitions/oauth.rb

```diff
@@ -29,7 +29,9 @@
     raise "Cannot restore OAuth: #{name}" unless @result[:success]
     # Check 'spec' first because pods only roll out when 'spec' had been changed
     if org_oauth_now['spec'] != org_oauth['spec']
-      step %Q/authentication successfully rolls out after config changes/
+    #  step %Q/authentication successfully rolls out after config changes/
+      timeout = 300
+      step %Q|operator "authentication" becomes available/non-progressing/non-degraded within #{timeout} seconds|
```

Comment:
We can't use only `step %Q|operator "authentication" becomes available/non-progressing/non-degraded within #{timeout} seconds|`, it does not ensure the operator really rotate successfully.
As you can see from your Runner/350548 log, it changes the configuration then **immediately** checks `oc get co authentication`, this check is too immediate, it only elapses 1 second, the output has not yet been changed.
If restoring fails, other cases will be affected. So we should use `step %Q/authentication successfully rolls out after config changes/` to ensure it first becomes Progress==True.

Reply:
Could you instead increase the timeout in the definition of `authentication successfully rolls out after config changes` in features/step_definitions/oauth.rb?

Reply:
300 seconds is long enough for the step. No need to fix this step. I know how to fix with my scenarios now, will update soon.